2_Functions_Part_2.ipynb

```diff
@@ -77,9 +77,20 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            new_list = new_list + lst
</pre><p>this is great, just to make sure you are aware - you could have used extend</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week05-sagin132/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='128'/>

Reply:
<p>Ok, cool!</p>